fuel-tx/src/transaction/validity.rs

```diff
@@ -112,6 +112,12 @@ impl Input {
                     recover_address()?
                 };
 
+                // This error is often returned during fuzzing when we encounter
+                // non-normalized signatures in `encode_signature`,
+                // in fuel-crypto/src/secp256/signature_format.rs.
+                // TODO(#815): Investigate if it makes sense to allow
+                //             non-normalized signatures during fuzzing.
+                #[cfg(not(fuzzing))]
```

Comment:
Hmm, I think we should always produce a normalized signatures. Otherwise, it is hard to use in production. Maybe @Dentosal has some ideas?

Reply:
Yeah, I think this in practice is very unlikely to ever happen - but the fact that we easily hit this in fuzzing warrants further investigation. I plan to look deeper into this in the context of https://github.com/FuelLabs/fuel-vm/issues/815

Reply:
Could you please look into https://github.com/FuelLabs/fuel-vm/issues/815 next?=) It would be nice to remove `#[cfg(not(fuzzing))]` before merging this PR

Reply:
Sure thing, I'll look into it=) Putting this PR on hold in the meantime.

Reply:
Done in https://github.com/FuelLabs/fuel-vm/pull/733/commits/ed5f70c8b62a4a3d8bceb9513e43bb3370ab95ba. I've removed the `#[cfg(not(fuzzing))]` now, and the fuzzer doesn't fail on those places anymore. The fix was as simple as bumping the `secp256k1` crate version.